infrastructure/cdn-in-a-box/traffic_ops_data/profiles/020-ATS_MID_TIER_CACHE.json

```diff
@@ -255,118 +255,176 @@
 			"configFile": "ip_allow.config",
 			"name": "location",
 			"secure": false,
-			"value": "/etc/trafficserver"
+			"value": "/opt/trafficserver/etc/trafficserver"
 		},
 		{
 			"configFile": "storage.config",
 			"name": "Drive_Prefix",
 			"secure": false,
 			"value": "/var/trafficserver/"
 		},
+		{
+			"configFile": "records.config",
+			"name": "CONFIG proxy.config.ssl.CA.cert.path",
+			"secure": false,
+			"value": "STRING /opt/trafficserver/etc/trafficserver/ssl"
+		},
+		{
+			"configFile": "records.config",
+			"name": "CONFIG proxy.config.ssl.client.CA.cert.path",
+			"secure": false,
+			"value": "STRING /opt/trafficserver/etc/trafficserver/ssl"
+		},
+		{
+			"configFile": "records.config",
+			"name": "CONFIG proxy.config.ssl.client.cert.path",
+			"secure": false,
+			"value": "STRING /opt/trafficserver/etc/trafficserver/ssl"
+		},
+		{
+			"configFile": "records.config",
+			"name": "CONFIG proxy.config.ssl.client.private_key.path",
+			"secure": false,
+			"value": "STRING /opt/trafficserver/etc/trafficserver/ssl"
+		},
+		{
+			"configFile": "records.config",
+			"name": "CONFIG proxy.config.ssl.ocsp.enabled",
+			"secure": false,
+			"value": "INT 1"
+		},
+		{
+			"configFile": "records.config",
+			"name": "CONFIG proxy.config.ssl.server.cert.path",
+			"secure": false,
+			"value": "STRING /opt/trafficserver/etc/trafficserver/ssl"
+		},
+		{
+			"configFile": "records.config",
+			"name": "CONFIG proxy.config.ssl.server.private_key.path",
+			"secure": false,
+			"value": "STRING /opt/trafficserver/etc/trafficserver/ssl"
+		},
+		{
+			"configFile": "records.config",
+			"name": "CONFIG proxy.config.ssl.server.ticket_key.filename",
+			"secure": false,
+			"value": "STRING NULL"
+		},
```

Comment:
Do these need to be added to the edge's profile as well?

Reply:
Nah, they were already there. With these added, the only configuration difference between the Edge and Mid profiles is that Edge is a reverse proxy:

```diff
--- 010-ATS_EDGE_TIER_CACHE.json	2021-05-18 12:21:57.791484259 -0600
+++ 020-ATS_MID_TIER_CACHE.json	2021-05-18 12:21:57.791484259 -0600
@@ -1,7 +1,7 @@
 {
 	"cdnName": "$CDN_NAME",
-	"description": "Edge Cache - Apache Traffic Server",
-	"name": "EDGE_TIER_ATS_CACHE",
+	"description": "Mid Cache - Apache Traffic Server",
+	"name": "MID_TIER_ATS_CACHE",
 	"routingDisabled": false,
 	"type": "ATS_PROFILE",
 	"params": [
@@ -93,7 +93,7 @@
 			"configFile": "records.config",
 			"name": "CONFIG proxy.config.reverse_proxy.enabled",
 			"secure": false,
-			"value": "INT 1"
+			"value": "INT 0"
 		},
 		{
 			"configFile": "records.config",
```